### PR TITLE
Completed setup work

### PR DESCRIPTION
**Pull request by Lockless404**
Added linters. Created item class. Setup the UI with app.rb and main.rb files.

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,102 @@
+class App
+  def run
+    puts 'Welcome to Catalog of Things!'
+    loop do
+      menu
+      option = input
+      break if option == '13'
```
Comment (Fikerte-T) [inline app.rb]: [] Input is an undefined variable. I think it should be `get.chomp`  and receive input from the user

File: item.rb [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  def label(label)
+    @label = label
+  end
+
+  def can_be_archived?
+    current = Date.today
+
+    difference_in_days = (@publish_date - current).to_i
+    (difference_in_days / 365.25) > 10
+  end
```
Comment (Fikerte-T) [inline item.rb]: -  Here the difference will be a negative integer. You can switch the operands to make the result positive.

 [ ] optional the year difference can be calculated and returned in one line
date.year - @publish_date.year > 10

**Review by Fikerte-T — CHANGES_REQUESTED** [review]
Hi @Lockless404 , 

Good job setting up the UI and implementing the methods.  
However, there are some issues that you still need to work on to go to the next milestone. 

## Required Changes ♻️


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**Review by Fikerte-T — APPROVED** [review]
Hi 👋 @Lockless404 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Comment by oluyaratosin123** [comment]
Hello @Lockless404 ,
I know your project has been approved but as a code checker I later examined that the app wasn't running properly, the column below shows the error
apprb

- [ ] Please go back to your `app.rb` file  **line-6** apply 
```Ruby 
option = gets.chomp.to_i 
```

File: app.rb [hunk 3]
```diff
@@ -0,0 +1,102 @@
+class App
+  def run
+    puts 'Welcome to Catalog of Things!'
+    loop do
+      menu
+      option = get.chomp
```
Comment (edwardoluyara) [inline app.rb]: Hello @Lockless404,

I know your project has been approved but as a code checker I later examined that the app wasn't running properly, the column below shows the error

## Optional : Required Changes ♻️

apprb

- [ ] Please go back to your `app.rb` file  **line-6** apply 
```Ruby 
option = gets.chomp.to_i 
```
Cheers and Happy coding!👏👏👏

**Review by edwardoluyara — CHANGES_REQUESTED** [review]

**Review by edwardoluyara — APPROVED** [review]

**Review by Fikerte-T — APPROVED** [review]
Hi 👋 Edward, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

**Pull request merged**